x-pack/plugins/ml/public/application/settings/anomaly_detection_settings.tsx

```diff
@@ -0,0 +1,229 @@
+}
+
+export const AnomalyDetectionSettings: FC<Props> = ({
+  canGetFilters,
+  canCreateFilter,
+  canGetCalendars,
+  canCreateCalendar,
+}) => {
+  const [calendarsCount, setCalendarsCount] = useState(0);
+  const [filterListsCount, setFilterListsCount] = useState(0);
+
+  const { toasts } = useNotifications();
+
+  useEffect(() => {
+    loadSummaryStats();
+  }, []);
+
+  async function loadSummaryStats() {
+    // Obtain the counts of calendars and filter lists.
+    if (canGetCalendars === true) {
+      try {
+        const calendars = await ml.calendars();
+        setCalendarsCount(calendars.length);
+      } catch (e) {
+        toasts.addDanger(
+          i18n.translate('xpack.ml.settings.anomalyDetection.loadingCalendarsCountErrorMessage', {
+            defaultMessage: 'An error occurred obtaining the count of calendars',
+          })
+        );
+      }
+    }
+
+    if (canGetFilters === true) {
+      try {
+        const filterLists = await ml.filters.filtersStats();
+        setFilterListsCount(filterLists.length);
+      } catch (e) {
+        toasts.addDanger(
+          i18n.translate('xpack.ml.settings.anomalyDetection.loadingFilterListCountErrorMessage', {
+            defaultMessage: 'An error occurred obtaining the count of filter lists',
+          })
+        );
+      }
+    }
+  }
+
+  return (
+    <Fragment>
+      <EuiPageContent className="mlSettingsPage__content" horizontalPosition="center">
+        <EuiPageContentHeader>
+          <EuiTitle>
+            <h2>
+              <FormattedMessage
+                id="xpack.ml.settings.anomalyDetection.anomalyDetectionTitle"
+                defaultMessage="Anomaly Detection"
+              />
+            </h2>
+          </EuiTitle>
+        </EuiPageContentHeader>
+
+        <EuiFlexGroup gutterSize="xl">
+          <EuiFlexItem grow={5}>
+            <EuiTitle size="s">
+              <h3>
+                <FormattedMessage
+                  id="xpack.ml.settings.anomalyDetection.calendarsTitle"
+                  defaultMessage="Calendars"
+                />
+              </h3>
+            </EuiTitle>
+            <EuiSpacer size="s" />
+            <EuiTextColor color="subdued">
+              <p>
+                <small>
+                  Calendars contain a list of scheduled events for which you do not want to generate
+                  anomalies, such as planned system outages or public holidays.
+                </small>
+              </p>
+            </EuiTextColor>
+            <EuiSpacer size="m" />
+            <EuiFlexGroup alignItems="center">
+              {canGetCalendars && (
+                <EuiFlexItem grow={false} style={{ display: 'block' }}>
+                  <EuiText>
+                    <FormattedMessage
+                      id="xpack.ml.settings.anomalyDetection.calendarsSummaryCount"
+                      defaultMessage="You have {calendarsCountBadge} {calendarsCount, plural, one {calendar} other {calendars}}"
```

Comment:
would be nice to show some loading state while calendars and filters data is loading

Reply:
I tried a loading spinner in place of the `EuiBadge`, but it didn't look great, so I have left this as-is. The counter is set to zero initially before the request returns with the actual counts.